docker/Dockerfile.dhtnode

```diff
@@ -0,0 +1,17 @@
+ARG DIST=xenial
```

Comment:
Can you put some comment explaining for each image and dockerfile which is the intention? Is a bit hard to not get lost with so many images being built, which one is for building the project, which one for running it or testing it.